managers/views.py

```diff
@@ -34,17 +38,20 @@ class ProjectsList(ListView):
     template_name = "managers/projects_list.html"
 
     def get_queryset(self) -> QuerySet:
+        logger.info(f"Get project query set for user: {self.request.user}")
```

Comment:
Not needed or debug.

Reply:
I made it debug, but in final decision I could delete it too :)